### ISSUE TITLE
Possible bug in datetime

### ISSUE THREAD

**Issue by Newman101** [joke2k/faker#513]
Hi. 

Looks like one of the unit tests is randomly failing: 

```
======================================================================
FAIL: test_date_time_between_dates_with_tzinfo (tests.factory.FactoryTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jremes/Projects/Python/faker-2/tests/factory.py", line 284, in test_date_time_between_dates_with_tzinfo
    self.assertTrue(datetime_start <= random_date)
AssertionError: False is not true

----------------------------------------------------------------------
Ran 80 tests in 12.309s
```

If needed, I can have a look at this issue. 

Thanks.

**Comment by fcurella** [comment]
Thanks for the report @Newman101 . If you have time to look into, I'd definitely appreciate it :)

**Comment by fcurella** [comment]
Closed in #524
